### PR TITLE
Allow webp images to be processed

### LINKED ISSUE #262: serving webp images to compatible browsers
Seems a logical extension. Anyone interested in doing that?

### PR DESCRIPTION
**Pull request by brittainmark**
Allow webp images
fixes #262

### CODE REVIEW

**Comment by lat9** [comment]
Solution for #262.  Thanks, @brittainmark; I'll review later today or in the morning.

**Comment by brittainmark** [comment]
All fine corruption in my test environment.

File: docs/image_handler/readme.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
 <body>
   <h1>Image Handler<sup>5</sup> for Zen Cart v1.5.7 (and later)</h1>
-  <h3>Version 5.3.2 by lat9, Zen4All, brittainmark, torvista, proseLA and brittainmark</h3>
+  <h3>Version 5.3.3 by lat9, Zen4All, brittainmark, torvista, proseLA and brittainmark</h3>
```
Comment (lat9) [inline docs/image_handler/readme.html:70]: Next version is already 5.3.2; remove duplicate of name!

**Review by lat9 — CHANGES_REQUESTED** [review]
Sorry it took so long to review!  Only changes are the minor ones to the readme.

**Comment by brittainmark** [comment]
No problem, I know you are busy. All done I hope.

**Review by lat9 — APPROVED** [review]

**Pull request merged**